plugins/c8y_configuration_plugin/src/upload.rs

```diff
@@ -393,70 +389,4 @@ mod tests {
 
         Ok(())
     }
-
-    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-    #[serial_test::serial]
-    async fn test_handle_config_upload_request() -> anyhow::Result<()> {
```

Comment:
Removed this test as it was redundant with `main::test_handle_config_upload_request_tedge_device` test.